user/homework.go

```diff
@@ -57,6 +59,36 @@ func (hw *Homework) Save() error {
 	return nil
 }
 
+// SaveAfterFetch filters out problems for tracks the user has yet to start
+// before saving them all to disk.
+func (hw *Homework) SaveAfterFetch(numArgs int) error {
+	hw.RejectUnstartedTracks(numArgs)
+	if len(hw.Items) == 0 {
+		fmt.Println(`
+You have yet to start a language track!
+View all available language tracks with "exercism tracks"
+Fetch exercises for your first track with "exercism fetch LANGUAGE"`)
+	}
```

Comment:
Instead of `LANGUAGE` this should be `TRACK_ID` (it's super confusing, I'm so sorry!).

Should we return that as an error instead of printing it here?